tests/test_routes.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+
+def test_get_every_single_planet(client, get_every_single_planet):
+    response = client.get(f"/planets")
+    response_body = response.get_json()
+
+    # assert
+    assert response.status_code == 200
+    assert len(response_body) == 5
+
+def test_get_all_planets_with_no_records(client):
+    # Act
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == []
+
+def test_get_one_planet_not_found_404(client):
+    # Arrange
+    # No setup needed - we want an empty database
+    # Act
+    response = client.get(f"/planets/10")
+    response_body = response.get_json()
+    # Assert
+    assert response.status_code == 404
+    assert response_body == {"message": "10 not found"}
+
+def test_get_one_planet_400(client, get_planet_invalid_id):
```

Comment:
Since we want to start the test with an empty database, the only fixture we need is the `client` so we can make calls to our API.
```py
def test_get_one_planet_400(client):
```